code/API_definitions/network_access_management.yaml

```diff
@@ -0,0 +1,2863 @@
+        - $ref: "#/components/parameters/netId"
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/NetworkUpdate"
+            examples:
+              NetworkUpdate:
+                $ref: "#/components/examples/NetworkUpdate"
+      responses:
+        "200":
+          description: Contains information about the updated network
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Network"
+              examples:
+                BaseNetwork:
+                  $ref: "#/components/examples/BaseNetwork"
+                NetworkWithDevices:
+                  $ref: "#/components/examples/NetworkWithDevices"
+        "400":
+          $ref: "#/components/responses/Generic400"
+        "401":
+          $ref: "#/components/responses/Generic401"
+        "403":
+          $ref: "#/components/responses/DefaultDeviceNotSupported403"
+        "404":
+          $ref: "#/components/responses/NetworkNotFound404"
+        "500":
+          $ref: "#/components/responses/Generic500"
+        "501":
+          $ref: "#/components/responses/Generic501"
+        "503":
+          $ref: "#/components/responses/Generic503"
+
+    delete:
+      tags:
+        - Isolated Networks
+      summary: Delete a network
+      operationId: deleteNetwork
+      description: |
+        Deletes the network and removes the configuration from any devices configured with the network that matches the given ID.
+
+        ## With Default Device
+        **Required Scopes:**
+          - **any of**
+            - `cpe-management-isolated-networks-write`
+            - `cpe-management-isolated-networks-primary-write`
+            - `cpe-management-isolated-networks-auxiliary-write`
+
+        ## Without Default Device
+        **Required Scopes:**
+          - `cpe-management-devices-write`
+          - **any of**
+            - `cpe-management-isolated-networks-write`
+            - `cpe-management-isolated-networks-primary-write`
+            - `cpe-management-isolated-networks-auxiliary-write`
+      parameters:
+        - $ref: "#/components/parameters/netId"
+      responses:
+        "204":
+          description: Successfully deleted the network
+        "400":
+          $ref: "#/components/responses/NetworkNotRemovable400"
+        "401":
+          $ref: "#/components/responses/Generic401"
+        "403":
+          $ref: "#/components/responses/DefaultDeviceNotSupported403"
+        "404":
+          $ref: "#/components/responses/NetworkNotFound404"
+        "500":
+          $ref: "#/components/responses/Generic500"
+        "501":
+          $ref: "#/components/responses/Generic501"
+        "503":
+          $ref: "#/components/responses/Generic503"
+
+  /isolated-networks/{netId}/devices:
+    parameters:
+      - $ref: "#/components/parameters/netId"
+    get:
+      tags:
+        - Devices

```

Comment:
```suggestion
```

Reply:
Keeping devices and removing networks since it is the resource returned.